src/damage.cpp

```diff
@@ -237,6 +237,11 @@ static const std::map<std::string, damage_type> dt_map = {
     { translate_marker_context( "damage type", "electric" ), DT_ELECTRIC }
 };
 
+std::map<std::string, damage_type> get_dt_map()
```

Comment:
```suggestion
const std::map<std::string, damage_type> &get_dt_map()
```
There is really no need to copy this.